learning/katas/go/Core Transforms/Map/ParDo struct/pkg/task/task.go

```diff
@@ -18,10 +18,7 @@ package task
 import "github.com/apache/beam/sdks/go/pkg/beam"
 
 func ApplyTransform(s beam.Scope, input beam.PCollection) beam.PCollection {
-	processFn := &multiplyByFn{
-		Factor: 5,
-	}
-	return beam.ParDo(s, processFn, input)
+	return beam.ParDo(s, &multiplyByFn{Factor: 5}, input)
```

Comment:
How about having the whole "&multiplyByFn{...}" be part of the answer placeholder?
![image](https://user-images.githubusercontent.com/5459430/80789407-8171fe00-8bbe-11ea-8ec6-e8ed8989ec2e.png)

Reply:
This seems to have an issue now. There is no placeholder at all. And the task description doesn't load properly.
Could it be because the directory name needs to be renamed to "ParDo Struct" with capital S?
![image](https://user-images.githubusercontent.com/5459430/80942383-3acb1080-8e17-11ea-9a25-6bded3455152.png)

Reply:
This looks good now! Thanks @damondouglas!